src/ConfigAggregator.php

```diff
@@ -285,12 +292,29 @@ private function cacheConfig(array $config, null|string $cachedConfigFile): void
         $this->writeCache($cachedConfigFile, $contents, $mode);
     }
 
+    /**
+     * @param list<PreProcessorCallable|class-string> $processors
+     * @param ProviderIterable $providers
+     * @return ProviderIterable
+     */
+    private function preProcessProviders(array $processors, iterable $providers): iterable
+    {
+        foreach ($processors as $processor) {
+            /** @var PreProcessorCallable $processor */
+            $processor = $this->resolveProcessor($processor);
+            $providers = $processor($providers);
```

Comment:
IMO to be inlined: `$processor` changes meaning here.

Reply:
Could you clarify what you mean by that? Do you mean from post-processor to pre-processor? Would you just rather a different name for clarity?

Reply:
Oh yeah, gotcha 👍, its a copy of the post processing method, I'll update that too